app/services/proofing/mock/state_id_mock_client.rb

```diff
@@ -72,8 +72,8 @@ def invalid_state_id_number?(state_id_number)
       end
 
       def invalid_state_id_type?(state_id_type)
-        !SUPPORTED_STATE_ID_TYPES.include?(state_id_type) ||
-          state_id_type.nil?
+        !SUPPORTED_STATE_ID_TYPES.include?(state_id_type) &&
+          !state_id_type.nil?
```

Comment:
This caused a lot of grief in tests. AFAICT, the non-mock client does _not_ require the presence of `state_id_type`. (See, e.g., null/empty-string responses in Cloudwatch.) The mock client shouldn't implement validation that real clients don't have and that does not serve a clear purpose.